cli/args.go

```diff
@@ -142,6 +142,18 @@ func filterTerraformExtraArgs(terragruntOptions *options.TerragruntOptions, terr
 	return out
 }
 
+func filterTerraformEnvVarsFromExtraArgs(terragruntOptions *options.TerragruntOptions, terragruntConfig *config.TerragruntConfig) map[string]string {
```

Comment:
This method will apply the env vars to every command. However, if `commands = ["apply"]` is specified, those env vars should only be added to the `apply` command. Could you do the command name check the same way as in `filterTerraformExtraArgs`?

Reply:
OK, got it :)